src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -218,6 +218,9 @@ const AppWrapper = (props: { children: ReactNode }) => {
       KeyStoreKeys.IDENTITY_ROOT_XPRV_KEY
     );
     const passwordIsSet = await checkKeyStore(KeyStoreKeys.APP_OP_PASSWORD);
+    const connectUrl = await Agent.agent.basicStorage.findById(
```

Comment:
Can we return the connectUrl from this function and use it in `initApp`? Since we are calling this findById a second time in initApp and we can remove that.

```
const { keriaConnectUrl } = await loadCacheBasicStorage();
if (keriaConnectUrl) {
...
```